lib/systemd/bridgehead@.service

```diff
@@ -2,14 +2,16 @@
 Description=Bridgehead (%i) Service
 
 [Service]
+User=bridgehead
 Restart=always
 RestartSec=30
 
 WorkingDirectory=/srv/docker/bridgehead/
 
-ExecStartPre=/bin/bash -c '`which docker-compose` -f %i/docker-compose.yml --env-file site-config/%i.env down'
-ExecStart=/bin/bash -c '`which docker-compose` -f %i/docker-compose.yml --env-file site-config/%i.env up'
-ExecStop=/bin/bash -c '`which docker-compose` -f %i/docker-compose.yml --env-file site-config/%i.env down'
+ExecStartPre=/srv/docker/bridgehead/stop-bridgehead.sh %i
+ExecStart=/srv/docker/bridgehead/start-bridgehead.sh %i
+RemainAfterExit=true

```

Comment:
Instead of `RemainAfterExit=true` you could also ensure that `start-bridgehead.sh` calls `docker-compose logs` at the end. This would have the side effect, that all logs of the containers would be searchable with `journalctl -u`.
On the otherhand you could change the type of this systemd unit to `Oneshot`.